src/sdbal/Vester.sol

```diff
@@ -0,0 +1,223 @@
+         }
+         _;
+     }
+
+     modifier onlyDaoMsig() {
+         if (msg.sender != DAO_MSIG) {
+             revert NotDAO();
+         }
+         _;
+     }
+
+     modifier onlyMaxisOps() {
+         if (msg.sender != MAXIS_OPS) {
+             revert NotMaxis();
+         }
+         _;
+     }
+
+     //////////////////////////////////////////////////////////////////
+     //                   Permissioned Setters                       //
+     //////////////////////////////////////////////////////////////////
+     function setBeneficiary(address _beneficiary) public onlyMaxisOps {
+         address oldBeneficiary = beneficiary;
+         beneficiary = _beneficiary;
+         emit BeneficiaryChanged(oldBeneficiary, _beneficiary);
+     }
+
+     //////////////////////////////////////////////////////////////////
+     //                       External functions                     //
+     //////////////////////////////////////////////////////////////////
+     /// @notice Get current vesting nonce. This nonce represents future vesting position nonce
+     /// @dev If needed to check current existing nonce, subtract 1 from this value
+     function getVestingNonce() external view returns (uint256) {
+         return vestingNonce;
+     }
+
+     /// @notice Get vesting position by nonce
+     /// @param _nonce Nonce of the vesting position
+     function getVestingPosition(uint256 _nonce) external view returns (VestingPosition memory) {
+         return vestingPositions[_nonce];
+     }
+
+     /// @notice Claim vesting position
+     /// @param _nonce Nonce of the vesting position
+     function claim(uint256 _nonce) external onlyBeneficiary whenNotPaused {
+         VestingPosition storage vestingPosition = vestingPositions[_nonce];
+         if (vestingPosition.claimed) {
+             revert AlreadyClaimed();
+         }
+         if (block.timestamp < vestingPosition.vestingEnds) {
+             revert NotVestedYet();
+         }
+         vestingPosition.claimed = true;
+
+         // Claim BAL rewards
+         ILiquidityGauge(address(SD_BAL_GAUGE)).claim_rewards(address(this), msg.sender);
+
+         // Transfer staked BAL to beneficiary
+         SD_BAL_GAUGE.safeTransfer(msg.sender, vestingPosition.amount);
+
+         emit Claimed(_nonce, vestingPosition.amount);
+     }
+
+     /// @notice Deposit logic but with default vesting period
+     /// @param _amount Amount of tokens to deposit
+     function deposit(uint256 _amount) external onlyMaxisOps whenNotPaused {
+         _deposit(_amount, DEFAULT_VESTING_PERIOD);
+     }
+
+     /// @notice Deposit logic
+     /// @param _amount Amount of tokens to deposit
+     /// @param _vestingPeriod Vesting period in seconds
+     function deposit(uint256 _amount, uint256 _vestingPeriod) external onlyMaxisOps whenNotPaused {
+         _deposit(_amount, _vestingPeriod);
+     }
+
+     /// @notice Maxis msig should be able to sweep any ERC20 tokens except staked aura bal
```

Comment:
Still a lot of mentions of AURA and AuraBAL here.  Both in comments and also  in code.  Makes it confusing to review.